xapian-core/cmake/xapian-config.cmake.in

```diff
@@ -1,5 +1,14 @@
 SET(prefix "@prefix@")
 SET(exec_prefix "@exec_prefix@")
-SET(XAPIAN_LIBRARIES "@libdir@/libxapian@LIBRARY_VERSION_SUFFIX@@SHLIBEXT@" CACHE FILEPATH "Libraries for Xapian")
+SET(XAPIAN_SHARED @XAPIAN_CMAKE_SHARED@ CACHE BOOL "Use shared xapian library")
+SET(XAPIAN_SHARED_LIBRARY "@libdir@/libxapian@LIBRARY_VERSION_SUFFIX@@SHLIBEXT@" CACHE FILEPATH "Shared Xapian library")
+SET(XAPIAN_STATIC_LIBRARY "@libdir@/libxapian@LIBRARY_VERSION_SUFFIX@.a" CACHE FILEPATH "Static Xapian library")
```

Comment:
Shouldn't this get the static library extension substituted?  MSVC uses `.lib` not `.a`.

Reply:
I don't know what changed in automake/libtool, but before it always created `.a` libraries, even when using MSVC.
I just tested it and this is not the case anymore. So I fixed this.